src/components/ButtonPanel.js

```diff
@@ -10,14 +10,21 @@ const keys = [
 ];
 
 const keyOperators = keys.map(row => (
-  <div key={row.toString()}>
-    { row.map(button => <Button button={button} key={button} />) }
+  <div className="row" key={row.toString()}>
+    { row.map(button => (
+      <Button
+        button={button}
+        key={button}
+        wide={button === '0'}
+        color={(button === '÷' || button === 'X' || button === '-' || button === '+' || button === '=') ? 'orange' : 'gray'}
```

Comment:
[OPTIONAL]
- Since you've already set the default value of `color` prop to orange in the `Button` component, there's no need for you to set it to orange here, you can just set it to gray if it's not any of these buttons

**_This is marked as optional because you may choose to implement it or not_**

Reply:
Optional change implemented.
Thank you for the feedback, very useful.